README.md

```diff
@@ -286,6 +286,19 @@ The following commands can be issued as batch commands (refer to [How to send ba
     ```
   
 ---
+### Available attributes/requirements
+The offer/bid functions accept attributes (dict attributes of the offer or bid) | requirements (list of requirements dicts that at least 1 of them need to be satisfied in the matching process)
+Currently supported attributes:
+- Offers:
+    - energy_type str: Energy type of the offer
+- Bids: No supported bids' attributes at the moment
+
+Currently supported requirements:
+- Offers:
+    - preferred_trading_partner List[str]: preferable area ids to match with.
+- Bids:
+    - energy_type List[str]: Energy types that the bid prefers to consume.
+    - preferred_trading_partner List[str]: preferable area ids to match with.
```

Comment:
Looking at the rendered HTML version, I think that the syntax could be changed for clarity. I'd also remove the types since the `README.md` is not supposed to be an exhaustive source of documentation (I think). For example:
```suggestion
- Offers:
    - `energy_type`: Energy type of the offer
- Bids: No supported bids' attributes at the moment

Currently supported requirements:
- Offers:
    - `preferred_trading_partner`: preferable area ids to match with.
- Bids:
    - `energy_type`: Energy types that the bid prefers to consume.
    - `preferred_trading_partner`: preferable area ids to match with.
```